src/components/facets/FacetList.vue

```diff
@@ -1,19 +1,37 @@
 <template>
-    <div v-if="!waitingForInitialSearch" class="hawk-facet-rail">
-        <div class="hawk-facet-rail__heading">{{ $t('Narrow Results') }}</div>
+  <div
+    v-if="!waitingForInitialSearch"
+    class="hawk-facet-rail"
+    :class="{ 'hawk-facet-rail__sticky': isNavSticky && isInResponsiveMode }"
+    @scroll="onScroll"
+  >
+    <div class="hawk-facet-rail__heading" @click="toggleFacetMobileMenu">
+      {{ $t("Filter By") }}
```

Comment:
This is a breaking change. We can change the label, but we have to update the translation object. Let's discuss it next week.